### PR TITLE
OTH-26 creating a separate zds group configuration, css and logo

### PR DESCRIPTION
**Pull request by edgarexpertcore**
creating zds group, fixing the zds group logo, fixing footer css position issue

### CODE REVIEW

File: web/sites/www.zds2.group.com/default.services.yml [hunk 1]
```diff
@@ -0,0 +1,174 @@
+parameters:
```
Comment (jcheek) [inline web/sites/www.zds2.group.com/default.services.yml]: The directory must be named zds.group .

File: web/themes/custom/ziquid_theme/css/styles.css [hunk 2]
```diff
@@ -75,6 +76,11 @@ h1 {
     border-bottom-right-radius: 24px;
 }
 
+#block-address  .text-formatted{
```
Comment (jcheek) [inline web/themes/custom/ziquid_theme/css/styles.css]: Please use drupal coding standards

File: web/sites/www.zds2.group.com/default.services.yml (hunk 1 above)
Comment (edgarexpertcore) [reply web/sites/www.zds2.group.com/default.services.yml]: done

File: web/themes/custom/ziquid_theme/css/styles.css (hunk 2 above)
Comment (edgarexpertcore) [reply web/themes/custom/ziquid_theme/css/styles.css]: done

**Pull request merged**